hadoop-hdds/common/src/main/java/org/apache/hadoop/ozone/OzoneConsts.java

```diff
@@ -351,7 +352,7 @@ private OzoneConsts() {
   public static final String GDPR_FLAG = "gdprEnabled";
   public static final String GDPR_ALGORITHM_NAME = "AES";
   public static final int GDPR_DEFAULT_RANDOM_SECRET_LENGTH = 16;
-  public static final String GDPR_CHARSET = "UTF-8";
+  public static final String GDPR_CHARSET = StandardCharsets.UTF_8.name();
```

Comment:
Both usages of `GDPR_CHARSET` are in calls to methods where both `String` and `Charset` versions exist, so we can make this change:

```suggestion
  public static final Charset GDPR_CHARSET = StandardCharsets.UTF_8;
```

Since it is only used in `GDPRSymmetricKey`, I would also consider inlining this constant.  I don't think we need to define it globally.

Reply:
I think keep `GDPR_CHARSET` here is better, because define constants about GDPR here.